crates/red_knot_python_semantic/src/types/property_tests.rs

```diff
@@ -285,6 +287,18 @@ mod stable {
         non_fully_static_types_do_not_participate_in_subtyping, db,
         forall types s, t. !s.is_fully_static(db) => !s.is_subtype_of(db, t) && !t.is_subtype_of(db, s)
     );
+
+    // All types should be assignable to `object`
+    type_property_test!(
+        all_types_assignable_to_object, db,
+        forall types t. t.is_assignable_to(db, KnownClass::Object.to_instance(db))
+    );
+
+    // And for fully static types, they should also be subtypes of `object`
+    type_property_test!(
+        all_fully_static_types_subtype_of_object, db,
+        forall types t. t.is_fully_static() => t.is_subtype_of(db, KnownClass::Object.to_instance(db))
```

Comment:
oops, you need a db here

```suggestion
        forall types t. t.is_fully_static(db) => t.is_subtype_of(db, KnownClass::Object.to_instance(db))
```

Reply:
Yes, I have it fixed locally, just working on your other comment, which revealed another bug 😆